### PR TITLE
docs(Supply Chain UI): :memo: Updating the Known Issues for the TDP -…

### PR DESCRIPTION
**Pull request by seagomezar**
… Supply Chain Plugin

This commit introduces documentation updates to address a known issue where the TAP GUI's Supply Chain visualization improperly expects configuration for data.packaging.carvel.dev for supply chains that do not generate Carvel packages. This behavior led to an error when choosing specific workloads within the Supply Chain tab of the TAP GUI. The updated documentation provides clarity on the error's nature and outlines a workaround involving RBAC configuration adjustments.

# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

File: release-notes.hbs.md [hunk 1]
```diff
@@ -144,6 +144,18 @@ enable TUF:
 
 - SCST - Store returns an expired certificate error message when a CA certificate expires before the app certificate. For more information, see [CA Cert expires](scst-store/troubleshooting.hbs.md#ca-cert-expires).
 
+#### <a id='1-9-0-ssc-ui-ki'></a> v1.9.0 Known issues: Supply Chain UI
+
+- When accessing the supply chain tab in the Tanzu Developer Portal, users might encounter an error related to data.packaging.carvel.dev. This issue arises even for supply chains that do not generate Carvel packages, indicating an unexpected configuration requirement. The error message displayed is related to permission issues and JSON parsing errors, specifically mentioning that the user "system:serviceaccount:tap-gui:tap-gui-viewer" cannot list resource "packages" in the API group "data.packaging.carvel.dev" at the cluster scope. Additionally, an unexpected non-whitespace character is reported after JSON at position 4.
```
Comment (chinigo) [inline release-notes.hbs.md:149]: **SUGGESTION**

Flip this sentence: "This issue indicates a misconfiguration and arises even for supply chains that do not generate Carvel packages."

File: release-notes.hbs.md [hunk 2]
```diff
@@ -144,6 +144,18 @@ enable TUF:
...
+
+Workaround: A temporary solution involves applying an RBAC configuration that includes permissions (get, watch, list) for the resources within the data.packaging.carvel.dev API group. This configuration mitigates the issue but it is highlighted that such a requirement should not be mandated for supply chains not generating Carvel packages.
+
+The error goes away after below rbac config is applied:
```
Comment (chinigo) [inline release-notes.hbs.md:153]: **SUGGESTION**

Suggest rephrasing to use more active verb: "Configuring RBAC to allow access to the Carvel package resource eliminates the error message:"

Also suggest adding the GVK section to this snippet, to more clearly indicate where this change needs to happen (i.e. in Kubernetes, not in Backstage):

```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
- apiGroups: [data.packaging.carvel.dev]
  resources: [packages]
  verbs: ['get', 'watch', 'list']
- … 
```

**Review by chinigo — APPROVED** [review]
Two optional suggestions but this is quite clear to me.

File: release-notes.hbs.md (hunk 1 above)
Comment (seagomezar) [reply release-notes.hbs.md]: Removed!

File: release-notes.hbs.md (hunk 2 above)
Comment (seagomezar) [reply release-notes.hbs.md]: Done!

**Comment by chinigo** [comment]
LGTM, thanks!

File: release-notes.hbs.md [hunk 3]
```diff
@@ -152,9 +152,27 @@ enable TUF:
 
 - SCST - Store returns an expired certificate error message when a CA certificate expires before the app certificate. For more information, see [CA Cert expires](scst-store/troubleshooting.hbs.md#ca-cert-expires).
 
+
+#### <a id='1-9-0-ssc-ui-ki'></a> v1.9.0 Known issues: Supply Chain UI
+
+- When accessing the supply chain tab in the Tanzu Developer Portal, users might encounter an error related to data.packaging.carvel.dev. The error message displayed is related to permission issues and JSON parsing errors, specifically mentioning that the user "system:serviceaccount:tap-gui:tap-gui-viewer" cannot list resource "packages" in the API group "data.packaging.carvel.dev" at the cluster scope. Additionally, an unexpected non-whitespace character is reported after JSON at position 4.
+
+Workaround: A temporary solution involves applying an RBAC configuration that includes permissions (get, watch, list) for the resources within the data.packaging.carvel.dev API group. This configuration mitigates the issue but it is highlighted that such a requirement should not be mandated for supply chains not generating Carvel packages.
```
Comment (kcoriordan) [inline release-notes.hbs.md:160]: Does  _**but it is highlighted that such a requirement should not be mandated for supply chains not generating Carvel packages**_ mean
This workaround should not be mandated for supply chains that do not generate Carvel packages.

or even
This workaround is only required for supply chains that generate Carvel packages.

Comment (seagomezar) [reply release-notes.hbs.md:160]: For removing the error it is required for all type of supply chains, even if they generate or not Carvel Packages. But that statement refers tois a know issue requesting that permision even when a supply chain does not generate Carvel packages.

Comment (chinigo) [reply release-notes.hbs.md:160]: Maybe

> Workaround: To mitigate against this problem, create an RBAC configuration that includes the `get`, `watch`, and `list` permissions for all resources within the `data.packaging.carvel.dev` API group. These permissions are overbroad — only supply chains that create Carvel packages really need this access — but granting them eliminates the error.

**Pull request merged**